css/styles.css

```diff
@@ -59,29 +162,58 @@ a {
 
 section {
   text-align: center;
-  margin: 6.5em auto;
+  margin: 80px auto;
+}
+
+h1,
+h2,
+h3,
+p {
+  color: rgba(23, 43, 77, 1);
 }
 
 h1 {
   font-size: 2.1em;
   font-weight: bold;
+  margin-top: 3em;
 }
 
 span {
-  color: #36b37e;
   display: block;
 }
 
+button {
+  background-color: #36b37f;
+  color: #fff;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 15px auto;
+}
+
 @media only screen and (max-width: 760px) {
+  #about-me > ul {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
```

Comment:
- [ ] As part of the project requirement, you are required to use `flexbox` to align the contents of the `about-me` Please resolve that. :slightly_smiling_face: 
- [ ] Your design also is not matching the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention the design  :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![F desi](https://github.com/flov3rh/portfolio/assets/43528594/75db6a53-bea7-4d0c-bfb0-0f35219264a1)

</td>
      <td>

![figma Des](https://github.com/flov3rh/portfolio/assets/43528594/64b8baf1-8947-4310-a08e-8e52a7be76be)

</td>
    </tr>
  </table>